### ISSUE TITLE
Drugs and buffs

### ISSUE THREAD

**Issue by MrJakobLaich** [kingarthur91/pyfood#2]
Conversation from discord so I don't lose it. 

Buffs that give the player movement speed for a specific time duration. 

Food gives buffs. 

eating should be possible to automate, so you don't have to eat bread every 10 mimutes. (FeederBot in gun slot and food in ammo slot)

Eating pyFood will give you some buffs
With better foods giving better buffs

(movement speed is critical in huge factories) 
As wesdotcool said: "I always turn up the movement speed in pymods using the console because the base is so big. But if you can just eat auog burgers early on in the game, then pymods solves the problems that it creates!!" 

Also, because animals should also be possible:
Some kind of Horse-Cart
That way you get some early Car Version if you have really big distances to travel + it's cheap

Feed the Horse steroids and it runs faster?

Oh, yeah, it would probably not be auog
You would probably have to make wheat and other stuff to get bread first and then later on switch to burgers and than pills or something

Because it's still "PY-food" and not just "food" xD

you would need some farms, some machines to grind the wheat into flour, some specialized furnaces, because a smeltery probably wouldn't produce good bread xD (also don't forget to slice the bread) 
BUT
You get to make bread eyes
And then you run faster :D

https://www.google.com/search?q=tomato+feeder&client=ms-android-huawei&prmd=isvn&source=lnms&tbm=isch&sa=X&ved=2ahUKEwim49GO3fnvAhUgg_0HHTJHAfsQ_AUoAXoECAIQAQ#imgrc=fs1rV6YEuDgKpM

early prototype of the FeederBot

Toasting bread needs sliced bread and some heatcoild which would need some kind of tungsten probably

Do we have tungsten in the pyMods yet?

**Comment by MrJakobLaich** [comment]
Okay the following idea should also be good I think

What if you could place restaurants as early game beacons? Because people with good food supply work faster, right?
At first I had specifically labs in mind, but what if we just assume that every machine has some people working inside? I know, Factorio is about a singular person that's stranded on an alien planet, but would be something to think abour

Now that I think about it, I always asumed that people work in the labs, well there I learn something new again.

Or maybe not beacons for the machines but just for the player? If the player is inside the range of a restaurant then he gains some movementspeed buff
That would 1) remove the need to eat stuff / automate that 2) give restaurants a use beside researching better food 3) probably look nice 4) would give the food stuff some reason to be actually automated instead of "well I have one machine that makes bread, I guess thats it with food"
Because!
If a restaurant consumes a constant strean of food and in turn gives you a specific boost, then you would probably also want to automate that and also increase the amount of boos that you recieve from some input (making better food)

I think I'm onto something here thinking

**Comment by MrJakobLaich** [comment]
TL;DR
restaurants take a constant stream of food and give the player some buff (movement speed) in some range (same as wirh beacons but a little bigger area and also only buffs the player) 
better food gives better buffs

**Comment by MrJakobLaich** [comment]
kitchen floor tiles
They will not increase your movement speed but should increase the effect from nearby restaurants
so in the later game you will gain a huge boost (necessary with big factory) and in the early game you don't have resources to make them anyways

**Comment by MrJakobLaich** [comment]
Regarding frozen food and other stuff:

i dont wanna put stabilizer E305 etc in my foods and give them a shelflife if i dont have a refrigerated warehouse ulricfood
